base_attachment_object_storage/models/ir_attachment.py

```diff
@@ -68,109 +68,114 @@ def _register_hook(self):
         if update_module:
             self.env['ir.attachment'].sudo()._force_storage_to_object_storage()
 
-    @api.model
-    def _save_in_db_domain(self):
+    @property
+    def _object_storage_default_force_db_config(self):
+        return {"image/": 50000, "application/javascript": 0, "text/css": 0}
```

Comment:
```suggestion
        return {"image/": 51200, "application/javascript": 0, "text/css": 0}
```
Otherwise a 48.9kB attachment will still be stored on object storage. This ensure it will be 50k as stated in the docs.

Reply:
I copied the previous configuration, but yes, it's better as you suggest, going to change